django/test/runner.py

```diff
@@ -494,7 +495,10 @@ class Shuffler:
 
     @classmethod
     def _hash_text(cls, text):
-        h = hashlib.new(cls.hash_algorithm)
+        if cls.hash_algorithm == 'md5':
+            h = md5(usedforsecurity=False)
+        else:
+            h = hashlib.new(cls.hash_algorithm)
```

Comment:
[`hashlib.new()`](https://docs.python.org/3/library/hashlib.html#hashlib.new) also accepts `usedforsecurity` as of Python 3.9. So I think it would be cleaner to eliminate the `md5`-specific `if` statement, and use a  `hashlib.new()` wrapper that passes `usedforsecurity` if available.

Reply:
Done